test/team_stats_test.rb

```diff
@@ -0,0 +1,178 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+require './lib/stat_parser'
+require 'pry'
+
+class TeamStatsTest < Minitest::Test
+
+    def setup
+      v_small_data_paths = {games: './data/game_very_small.csv',
+                            teams: './data/team_info.csv',
+                            game_teams: './data/game_teams_stats_very_small.csv'}
+
+      small_data_paths = {games: './data/game_small.csv',
+                          teams: './data/team_info.csv',
+                          game_teams: './data/game_teams_stats_small.csv'}
+
+      data_paths = {games: './data/game.csv',
+                    teams: './data/team_info.csv',
+                    game_teams: './data/game_teams_stats.csv'}
+
+      @v_small_data = StatTracker.from_csv(v_small_data_paths)
+      @small_data = StatTracker.from_csv(small_data_paths)
+      @data = StatTracker.from_csv(data_paths)
+    end
+
+  def test_team_info_with_respective_team_attributes
+    expected = {
+                team_id: 3,
+                franchise_id: 10,
+                short_name: "NY Rangers",
+                team_name: "Rangers",
+                abbreviation: "NYR",
+                link: "/api/v1/teams/3"
```

Comment:
At the moment, I believe that the ONLY info we have saved for teams are: `team_id` and `team_name` -- It looks like we should probably save some more.

Reply:
Yes, we realized when Brian posted the Spec Harness that they did want to see the full team info for this one. I took a stab at updating the conversions module (and StatParser/conversions_test) to no longer delete these keys and sent you a Pull Request!